core/store/store.go

```diff
@@ -34,3 +34,28 @@ type KVStore interface {
 
 // Iterator is an alias db's Iterator for convenience.
 type Iterator = dbm.Iterator
+
+type CoreIterator interface {
```

Comment:
I think this should be renamed `Iterator`.

Reply:
will do, wanted to get sign off before changing it cause I will need to modify the implementation of the interface